ui/src/components/DownloadImageForm.tsx

```diff
@@ -0,0 +1,82 @@
+import { FC } from "react";
+import { Form } from "react-bootstrap";
+import { FormikProps } from "formik/dist/types";
+import { DownloadImageFormValues } from "types/token";
+
+interface Props {
+  formik: FormikProps<DownloadImageFormValues>;
+}
+
+const DownloadImageForm: FC<Props> = ({ formik }) => {
+  return (
+    <div>
+      <Form noValidate>
+        <Form.Group className="mb-3" controlId="type">
+          <Form.Label>Image type</Form.Label>
+          <Form.Check
+            type="radio"
+            label="iso"
+            name="type"
+            value="iso"
+            checked={formik.values.type == "iso"}
+            onChange={formik.handleChange}
+          />
+          <Form.Check
+            type="radio"
+            label="raw"
+            name="type"
+            value="raw"
+            checked={formik.values.type == "raw"}
+            onChange={formik.handleChange}
+          />
+        </Form.Group>
+        <Form.Group className="mb-3">
+          <Form.Label>Installation target</Form.Label>
+          <Form.Check
+            type="checkbox"
+            label="Wipe the target drive"
+            name="install.force_install"
+            checked={formik.values.install.force_install}
+            onChange={formik.handleChange}
+            onBlur={formik.handleBlur}
+          />
```

Comment:
I can't tell what the default state is, this one should be ticked by default.

Reply:
Fixed.